Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -1546,16 +1546,21 @@ public struct RenderNodeTranslator: SemanticVisitor {
             
             // Children of the current symbol that have not been curated manually in a task group will all
             // be automatically curated in task groups after their symbol kind: "Properties", "Enumerations", etc.
-            let alreadyCurated = Set(sections.flatMap { $0.identifiers })
             let groups = try! AutomaticCuration.topics(for: documentationNode, withTraits: allowedTraits, context: context)
             
-            sections.append(contentsOf: groups.compactMap { group in
-                let newReferences = group.references.filter { !alreadyCurated.contains($0.absoluteString) }
-                guard !newReferences.isEmpty else { return nil }
-                
+            for group in groups {
+                let newReferences = group.references
                 contentCompiler.collectedTopicReferences.append(contentsOf: newReferences)
-                return TaskGroupRenderSection(taskGroup: (title: group.title, references: newReferences))
-            })
+
+                // If the section has been manually curated, merge the references of both the automatic curation and the manual curation into one section (rdar://61899214).
+                if let duplicateSectionIndex = sections.firstIndex(where: { $0.title == group.title }) {
```

Comment:
This exact string comparison might be too strict. For example, I tried creating a topic group like this:

```
### Type properties

- ``!=(_:_:)``
```

(lower case "p") ... and my docs ended up with redundant topic sections. Can we use a case insensitive check? And possibly even remove duplicate whitespace? The same thing happened when added two spaces between "Type" and "Properties"... to do that we might need a regex of some kind.

Might be overkill?

Reply:
I thought about this but in the end decided to go with the simplest solution for the moment (no fuzzy matching). Even though I think it would be helpful for the user in some cases, I'm leaning towards no because it's hard to find a level of leniency that would work for all use cases -- e.g. if someone has a topic section "Initialisers" (s instead of z), should we also merge those? And also what you've mentioned about multiple spaces.

WDYT?

Reply:
I actually had second thoughts about this too - I think going with the most transparent approach makes the most sense. If an author sees a duplicate topic section because of capitalization or spelling, then it should be obvious how to fix it, and also the tool's behavior is easier to understand.